containers/container.yml

```diff
@@ -110,7 +110,7 @@ services:
 registries:
   oc-cluster:
     url: 172.30.1.1:5000
-    namespace: foreman
+    namespace: "{{ project_name }}"
```

Comment:
I don't think this is really the same playbook.

I think this is more ansible-container build time than deployment.

Not really sure what we should do for this namespace to make to configurable.